src/app/timetable/TubeService.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import { withStyles } from '@material-ui/core/styles'
+import List from '@material-ui/core/List'
+import ListItem from '@material-ui/core/ListItem'
+import ListItemText from '@material-ui/core/ListItemText'
+import Divider from '@material-ui/core/Divider'
+
+// Agh why is this erroring and not showing on the browser!!!!!!!!
+const styles = theme => ({
+  root: {
+    width: '100%',
+    maxWidth: '360px',
+    backgroundColor: theme.palette.background.paper,
+  },
+})
+
+export function ListDividers(props : ) {
```

Comment:
Define your `interface Props` above this then tell typescript the type of `props`